Sources/PkgConfig/Module+PkgConfig.swift

```diff
@@ -82,3 +83,40 @@ private extension SystemPackageProvider {
         return providers.filter{ $0.isAvailable }.first
     }
 }
+
+private struct FlagsFilter {
+    let flag: String
+    let hasValue: Bool
+}
+
+private let includePath = FlagsFilter(flag: "-I", hasValue: true)
+private let linkPath = FlagsFilter(flag: "-L", hasValue: true)
+private let linkFlag = FlagsFilter(flag: "-l", hasValue: false)
```

Comment:
This isn't correct, actually the compiler driver will accept all forms of these arguments with and without a space between the value. This should make the code simpler.

Also, we should probably accept `-F` (framework search path) and `-framework` even though they probably aren't used much.